File: snippets/rs/array/empty/empty.rs
fn main()
{
    let mut array = vec![];
    if array.is_empty() { println!("array is empty") };    // ≡
    array.push(100);
    if !array.is_empty() { println!("array has items") };  // ≡
    array.clear();
    if array.is_empty() { println!("array is empty again") };
}

File: snippets/php/array/empty/empty.php
<?php
$arraay = [];
if (empty($arraay)) echo "array is empty\n";    // ≡
$arraay[] = 100;  // add an item
if (!empty($arraay)) echo "array has items\n";  // ≡
array_splice($arraay, 0);  // clear the array
if (empty($arraay)) echo "array is empty again\n";
?>